frontend/libs/studio-components/src/components/StudioPageHeader/StudioPageHeaderLeft/StudioPageHeaderLeft.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { type ReactNode, type ReactElement } from 'react';
+import { DigdirLogoLink } from './DigdirLogoLink';
+
+export type StudioPageHeaderLeftProps = {
+  children?: ReactNode;
+  title?: string;
+};
```

Comment:
Does it make sense not to provide title and children? Should be make at least one of the props required? Something like this:

```suggestion
export type StudioPageHeaderLeftProps = 
  | { children: ReactNode; title?: string }
  | { children?: ReactNode; title: string };
```

Reply:
I agree, but this will also make the title always show next to the logo in the header. I have now changes the code to work better 😄